packages/@lwc/compiler/src/rollup-plugins/minify.ts

```diff
@@ -13,6 +12,8 @@ import { NormalizedOutputConfig } from '../options';
  * Rollup plugin applying minification to the generated bundle.
  */
 export default function({ sourcemap }: NormalizedOutputConfig): Plugin {
+    // [perf optimization] inline the import to prevent node-tool from parsing unused lib until compiling for 'prod'.
```

Comment:
Remove the comment here, or remove the refence to `node-tool`.

```suggestion
    // Inlining the `terser` module require to only pay the parsing and evaluation cost for modules that need to 
    // compiled in `production` mode.
```